src/main/webapp/js/instructorFeedbackEdit/questionContrib.es6

```diff
@@ -23,6 +23,9 @@ function setDefaultContribQnVisibilityIfNeeded(questionNum) {
     $currentQuestionTable.find('input.visibilityCheckbox')
                          .filter('.recipientCheckbox')
                          .filter('[value="INSTRUCTORS"],[value="RECEIVER"]').prop('checked', true);
+    // Set dropdown button text
+    $currentQuestionTable.find('div.visibility-options-dropdown button.dropdown-toggle')
+                         .html('Please select a visibility option <span class="caret"></span>');
```

Comment:
Use the method `setVisibilityDropdownMenuText` in `visibilityOptions.es6` instead. But perhaps we should wait for #7522 to be merged first? Massive migration to webpack.